Filters/Transform/MosaicTransformFilter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhotoEnhancer
+{
+    public class MosaicTransformFilter : ParametrizedFilter<EmptyParameters>
+    {
+        MosaicTransformer transformer;
+
+        public MosaicTransformFilter(string name, 
+            MosaicTransformer transformer)
+        {
+            this.name = name;
+            this.transformer = transformer;
+        }
```

Comment:
Зачем передавать трансформер в конструктор, если вы уже зафиксировали тип трансформера? Проще его создавать в конструкторе.
```suggestion
        public MosaicTransformFilter(string name)
        {
            this.name = name;
            this.transformer = new MosaicTransformer();
        }
```